nym-wallet/src-tauri/Cargo.toml

```diff
@@ -49,7 +49,7 @@ features = ["ts-rs"]
 
 [dev-dependencies.validator-client]
 path = "../../common/client-libs/validator-client"
-features = ["ts-rs"]
+features = ["typescript-types"]
```

Comment:
Is there still a GitHub Action that builds the types? And would this affect it?

Reply:
The type building is done via simple `cargo test` command which is still done as part of our CI pipeline and hence it should be unaffected.